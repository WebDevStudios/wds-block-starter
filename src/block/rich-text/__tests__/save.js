/* global expect, test */

/**
 * External dependencies
 */
import { create as createTestRenderer } from 'react-test-renderer';

/**
 * WordPress dependencies
 */
import { RichText } from '@wordpress/block-editor';

test(
	'adds 1 + 2 to equal 3',
	() => {
		expect(
			1 + 2
		).toBe( 3 );
	}
);

// @see https://jestjs.io/docs/en/tutorial-react
test( 'renders RichText from WordPress components', () => {
	const testRenderer = createTestRenderer( <RichText.Content value="hi" /> );

	expect( testRenderer.toJSON() ).toMatchSnapshot();
} );
