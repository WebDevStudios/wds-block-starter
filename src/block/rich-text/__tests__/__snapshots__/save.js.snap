// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RichText from WordPress components 1`] = `
<div
  dangerouslySetInnerHTML={
    Object {
      "__html": "hi",
    }
  }
/>
`;
